connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -141,11 +141,37 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
   );
 
   await crawler.run([webCrawlerConfig.url]);
+  await crawler.teardown();
+
+  // Adjusting page parent urls.
```

Comment:
I thiiiink we can do it incrementally:
- Inserting a new WebCrawlFolder: check for existing pages and update their parents to it.
- Inserting a page, check for existing WebCrawlFolder parent and update.